CONTRIBUTING.md

```diff
@@ -23,9 +25,76 @@ pnpm install
 pnpm build
 ```
 
-## Testing
+# Developing
+
+For building everything in watch-mode, run:
+
+```sh
+# build all projects in watch-mode
+pnpm dev
+```
+
+File watching is done by `nodemon` for increased performance.
+
+Check `nodemon.config.json` for all settings.
+
+> **Note**: You can `pnpm dev` a single package:
+>
+> ```sh
+> cd packages/abi-coder
+> pnpm dev
+> ```
+
+# Using local sym-linked packages
+
+First, we need to link our `fuels` package globally in our local `global pnpm store`:
+
+```sh
+cd fuels-ts/packages/fuels
+pnpm link --global
+```
+
+Let's check it out:
+
+```sh
+pnpm list --global
+```
+
+Cool, now on the root directory of `my-local-project`:
+
+```sh
+cd my-local-project
+pnpm link --global fuels
+```
+
+That's it — `my-local-project` is now using our local version of `fuels`.
+
+The same can be done with all other packages:
+
+```sh
+cd fuels-ts/packages/wallet
+pnpm link --global
+
+# ...
+
+pnpm list --global # validating
+
+# ...
+
+cd my-local-project
+pnpm link --global @fuel-ts/wallet
+```
+
+> **Warning** When using your local version of `fuels`, remember that you need to run `pnpm build` on every change you do to source files, otherwise the changes won't be propagated to dependent apps using it. More often, you'll want to use `pnpm dev`.
```

Comment:
I'm not quite sure I understand this section of the documentation properly. Could you clarify it for me, please?

Would running `pnpm dev` work with symlinks? Or is the only way to use symlinks is to build the files each time we make a change using `pnpm build`?

Reply:
https://github.com/FuelLabs/fuels-ts/pull/984/commits/5b565aa81dc2c180fb4e27f342565538c44d0506